Formula/git-gitlab.rb

```diff
@@ -0,0 +1,46 @@
+require "language/go"
+
+class GitGitlab < Formula
+  desc "Git CLI support for GitLab"
+  homepage "https://github.com/numa08/git-gitlab/tree/9a1a74a7609178f25afc1786e01c2375762de48c"
+  url "https://github.com/numa08/git-gitlab/archive/9a1a74a7609178f25afc1786e01c2375762de48c.tar.gz"
```

Comment:
We need a tagged, stable release from upstream here. We won't accept the latest commit, even in tarball form. Most upstreams are pretty willing to tag releases if asked, so that's something to consider.